week-01/15.三数之和.py

```
@@ -0,0 +1,33 @@
+#
+# @lc app=leetcode.cn id=15 lang=python3
+#
+# [15] 三数之和
+#
+
+# @lc code=start
+class Solution:
+    def threeSum(self, nums: List[int]) -> List[List[int]]:
```

Ruff

Undefined name `List`

(F821)